code/modules/hydroponics/grown/peas.dm

```diff
@@ -0,0 +1,106 @@
+	wine_power = 50
+	wine_flavor = "It tastes, distressingly, of fermented peas."
+	// distill_regeant = /datum/reagent/saltpetre //if allowed, remove wine_power, ya dingus.
+
+
+// Laughin' Peas
+/obj/item/seeds/peas/laugh
+	name = "pack of laughin' peas"
+	desc = "These seeds give off a very soft purple glow.. they should grow into Laughin' Peas."
+	icon_state = "seed-laughpeas"
+	species = "laughpeas"
+	plantname = "Laughin' Peas"
+	product = /obj/item/reagent_containers/food/snacks/grown/laugh
+	maturation = 7
+	potency = 10
+	yield = 7
+	production = 5
+	growthstages = 3
+	growing_icon = 'icons/obj/hydroponics/growing_vegetables.dmi'
+	icon_grow = "laughpeas-grow"
+	icon_dead = "laughpeas-dead"
+	genes = list (/datum/plant_gene/trait/repeated_harvest, /datum/plant_gene/trait/glow/purple)
+	mutatelist = list (/obj/item/seeds/peas/laugh/peace)
+	reagents_add = list (/datum/reagent/consumable/laughter = 0.05, /datum/reagent/consumable/sugar = 0.05, /datum/reagent/consumable/nutriment = 0.07)
+	rarity = 25 //It actually might make Central Command Officials loosen up a smidge, eh?
+
+
+/obj/item/reagent_containers/food/snacks/grown/laugh
+	seed = /obj/item/seeds/peas/laugh
+	name = "pod of laughin' peas"
+	desc = "Ridens Cicer, guaranteed to improve your mood dramatically upon consumption!"
+	icon_state = "laughpeas"
+	filling_color = "#ee7bee"
+	bitesize_mod = 2
+	foodtype = VEGETABLES
+	tastes = list ("a prancing rabbit" = 1) //Vib Ribbon sends her regards.. wherever she is.
+	wine_power = 90
+	wine_flavor = "a vector-graphic rabbit dancing on your tongue"
+
+
+// World Peas - Peace at last, peace at last...
+/obj/item/seeds/peas/laugh/peace
+	name = "pack of world peas"
+	desc = "These rather large seeds give off a soothing blue glow..."
+	icon_state = "seed-worldpeas"
+	species = "worldpeas"
+	plantname = "World Peas"
+	product = /obj/item/reagent_containers/food/snacks/grown/peace
+	maturation = 20
+	potency = 75
+	yield = 1
+	production = 10
+	growthstages = 3
+	growing_icon = 'icons/obj/hydroponics/growing_vegetables.dmi'
+	icon_grow = "worldpeas-grow"
+	icon_dead = "worldpeas-dead"
+	genes = list (/datum/plant_gene/trait/glow/blue)
+	reagents_add = list (/datum/reagent/pax = 0.15, /datum/reagent/drug/happiness = 0.1, /datum/reagent/consumable/nutriment = 0.15)
+	rarity = 50 // This absolutely will make even the most hardened Syndicate Operators relax, and lay down their arms.
+
+
+/obj/item/reagent_containers/food/snacks/grown/peace
+	seed = /obj/item/seeds/peas/laugh/peace
+	name = "cluster of world peas"
+	desc = "A plant discovered through extensive genetic engineering, and iterative graft work. It's rumored to bring peace to any who consume it. In the wider AgSci community, it's attained the nickname of 'Pax Mundi'." //at last... world peas. I'm not sorry.
+	icon_state = "worldpeas"
+	filling_color = "#0099CC"
+	bitesize_mod = 4
+	foodtype = VEGETABLES
+	tastes = list ("numbing tranquility" = 2, "warmth" = 1)
+	wine_power = 100
```

Comment:
why so high on the wine power?

Reply:
Well, I figured something that brings near instant happiness and a feeling of peace would likely distill into a potent drink that'd numb you to the world. You can change it if you want, but I don't think it's that strong, compared to what Strange Seeds, or Bacchus Blessing provides.

Reply:
i think 30u is lethal is all iirc could be wrong

Reply:
That would be at wine_power 300, which would put it on par with Bacchus Blessing. At wine_power 100, it'd be comparable to Manly Dorf, but vastly less producible at round start. It should take 100ish units to kill a non-traited person. I could be reading the code for fermentation wrong, but I'm relatively sure it directly converts the wine_power into boozepwr when it ferments.

Reply:
Ye your right im misstaken

Reply:
being fixated on boozepwr seems odd considering there are easier and better ways to kill people in botany anyways